loam/allocation.lisp

```diff
@@ -231,6 +231,128 @@
 
 (defun ptr-wide-tag (ptr) (widen (ptr-tag ptr)))
 
+#+nil
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  ;; TODO: Any better way to do this?
+  (defun concat-sym (root suf)
+    (intern (format nil "~A~A" root suf))))
+
+;; TODO: Maybe it would be better for this to  belong here? I've copied this over to evaluation.lisp for now.
+#+nil
+(defmacro defmem (prog-name superclasses config &body arg-specs)
```

Comment:
Whatever we decide, let's have it live in only one place rather than duplicate and comment.

Reply:
I've deleted